src/plot_api/plot_api.js

```diff
@@ -3802,6 +3802,7 @@ function makePlotFramework(gd) {
         .classed('gl-container', true);
 
     fullLayout._paperdiv.selectAll('.main-svg').remove();
+    fullLayout._paperdiv.selectAll('.modebar-container').remove();
```

Comment:
`.select` should suffice here, as there's only one `<div.modebar-container>` per graph.

Reply:
Done in 567eddedd5ef3e1fd9e45ee0c03a6cea5d279d20